src/export_json.lean

```diff
@@ -340,20 +340,52 @@ do docs ← olean_doc_strings,
      (filename, json.array $ l.map write_module_doc_pair) :: rest
    end) [])
 
-meta def get_instances : tactic (rb_lmap string string) :=
+/-- Extract `[foo, bar]` from `has_pow foo bar`.
+
+The result is a map containing the type names as keys.  -/
+meta def find_instance_types (e : expr) : tactic (rb_map string unit) :=
+list.mfoldl (λ m e, do
+  e ← dunfold [`decidable_rel, `decidable_pred] e {fail_if_unchanged := ff},
```

Comment:
How does this even happen?  `e` is an *argument* after all.

Reply:
I no longer remember, I think this must be vestigial from an earlier version